<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Poster</class>
 <widget class="QWidget" name="Poster">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Poster{
background-color: black;
}</string>
  </property>
  <widget class="QLabel" name="PosterBackArrowLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel:hover{
background-color: rgb(99, 110, 114);
}</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Back-Arrow/left-arrow.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="PosterTab">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>1481</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>-1</number>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../Images/Back-Arrow.qrc"/>
  <include location="../Images/TwitterLogo.qrc"/>
 </resources>
 <connections/>
</ui>
